lib/services/messenger/translators/flowdock.ts

```diff
@@ -50,12 +50,12 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 		const inject = metadata ? metadata : '';
 		const last = footer ? `\n${footer}` : '';
 		if ((first.length + inject.length + body.length + last.length) < lengthLimit) {
-			return `${first}${inject}${body}${last}`;
+			return `${first}${body}${last}${inject}`;
```

Comment:
This is the perfect opportunity to add some unit tests for this pure function :)

Reply:
Done